### PR TITLE
added Doors Data and its properties

### PR DESCRIPTION
**Pull request by krishnabangalore**
jira issue: wp-614

Doors Data for Vehicle API and its Properties pushed 

@Habib: Please check for package.json and webinos_config.json whether it is correct or not since the git status showed the files are changed and needs a commit.

### CODE REVIEW

**Comment by andreasbotsikas** [comment]
Please remove changes in webinos_config.xml and package.json. You can't just change the whole file (changing indentation) as it is impossible to verify that they are right. When you commit your changes, _always_ unselect those files, unless you have made a required change.

**Comment by krishnabangalore** [comment]
will do that

**Comment by krishnabangalore** [comment]
Andreas I have updated it to the current files. Please have a look and let me know if it's not correct.

File: webinos_config.json [hunk 1]
```diff
@@ -102,7 +102,9 @@
     },
     {
       "name": "devicestatus",
-      "params": {}
+      "params": {
```
Comment (andreasbotsikas) [inline webinos_config.json:105]: Unfortunately your base is older than the one in the current master and I am afraid this won't automerge nicely. Can you do a rebase discarding your changes in order to not modify the webinos_config with your pr?

Comment (andreasbotsikas) [inline webinos_config.json:105]: I mean discarding the changes in the webinos_config file only...

File: package.json [hunk 2]
```diff
@@ -47,7 +47,8 @@
   , "passport": "x.x.x"
   , "passport-yahoo": "x.x.x"
   , "passport-google": "x.x.x"
-  , "serialport2": "0.0.5"
+  , "open": "x.x.x"
```
Comment (andreasbotsikas) [inline package.json:50]: Are these changes related to your PR? I think this is also an issue of rebase. Please confirm and do a rebase discarding this files changes too
